ui/application-window.ui

```diff
@@ -38,6 +38,51 @@ Author: Philipp Wolfer <ph.wolfer@gmail.com>
     <property name="can_focus">False</property>
     <property name="icon_name">media-playback-stop-symbolic</property>
   </object>
+  <object class="GtkPopover" id="pop_menu">
+    <property name="can_focus">False</property>
+    <child>
+      <object class="GtkBox">
```

Comment:
Items are currently aligned to the center, when they should be aligned to the left for Primary Menus

Reply:
Fixed